lib/small_url_web/live/stats_live.html.leex

```diff
@@ -33,4 +33,20 @@
         <span class="text-gray-400"><%= format_date(@link_info.last_click_date) %></span>
       </div>
     </div>
+
+    <div class="mt-10 bg-white max-w-full w-full p-10 rounded-md shadow">
+      <h1 class="text-black mb-5 font-bold">Last 30 days</h1>
+      <div class="p-4 max-w-full overflow-y-auto tiny-scrollbar">
+        <%= 
+          spec = Vl.new(width: 600, height: 400)
+            |> Vl.data_from_values(@click_frequencies)
+            |> Vl.mark(:line)
+            |> Vl.encode_field(:x, "month", field: "month", type: :nominal, title: "days")
+            |> Vl.encode_field(:y, "clicks", field: "clicks",  type: :quantitative, title: "Number of clicks in the last 30 days")
+            |> Vl.to_spec
+
+            live_component(SmallUrlWeb.Output.VegaLiteStaticComponent, id: 10, spec: spec)
```

Comment:
We can call render_output here instead of using the liveview directly.

Reply:
Or we can instead remove the render_output function entirely, since it's not rendering anything inside the actual liveview module.